conans/test/util/vcvars_clangcl_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+
+import platform
+import unittest
+from nose.plugins.attrib import attr
+from conans.model.settings import Settings
+from conans.client.conf import default_settings_yml
+from conans import tools
+
+
+@attr('visual_studio')
+@unittest.skipUnless(platform.system() == "Windows", "Requires Windows")
+class VCVarsClangClTest(unittest.TestCase):
+
+    def test_simple(self):
+        if platform.system() != "Windows":
+            return
+
+        settings = Settings.loads(default_settings_yml)
+        settings.compiler = 'clang'
+        settings.compiler.version = '5.0'
+        settings.compiler.libcxx = 'libstdc++'  # FIXME: this is needed to be set to something, add msstl variant
```

Comment:
Why is this needed for vcvars to work? I read the original issue but did not get it

Reply:
I've just checked and it's actually non needed lol :)
it's needed in case I am running conan from command line only, I believe
I'll remove this

Reply:
done